documentation/airport-format.md

```diff
@@ -1,309 +1,228 @@
-    }
-  },
-  "stars": {  // contains all STARS available at this airport
-    "PYE1" : {
-      "icao": "PYE1",               // (req) ICAO identifier for SID (this is NOT the full name, always 2-6 characters)
-      "name": "Point Reyes One" ,   // (req) Name of SID as it would be said aloud (it is used by speech synthesis to pronounce "OFFSH9")
-      "suffix": {"1L":"", "1R":"", "28L":"", "28R":""},   // (optional) defines suffixes to STAR name based on runway (eg '7W' for 'MIKOV 7W').
-                                                          // Common for European-style STARs. If not needed (like in USA), leave this part out.
-      "entryPoints": {    // (optional) Defines entryPoints for a given SID. Common for FAA-style (USA) SIDs. If not needed (like in Europe), leave this part out.
-          "ENI": ["ENI"], // defines the "OFFSH9.SNS" transition as being a single fix, "SNS". Is often a list instead.
-          "MXW": ["MXW"]  // Note that this connects to the end of previous sections, so an example route: SEPDY->ZUPAX->EUGEN->SHOEY->BSR
-      },
-      "body": ["PYE", ["STINS", "A230|S250"], "HADLY"], // (optional) This is where you store the waypoints when all segments are along the same path
-      "rwy": {  // (optional) For runway-transitions (eg "descending via HAWKZ4, landing north")
-          "1L" : [["SEPDY", "A19+"], "ZUPAX"],  // List fixes here that are specific to a particular runway configuration.
-          "1R" : [["SEPDY", "A19+"], "ZUPAX"],  // If any runways are listed, all must be listed.
-          "28L": [["SENZY", "A25+"], "ZUPAX"],
-          "28R": [["SENZY", "A25+"], "ZUPAX"]
-      },
-      "draw": [["ENI*","PYE"], ["MXW*","PYE"], ["PYE","STINS","HADLY","OSI"]]
-        // (req) This "draw" section is what defines how the SID is to be drawn on the scope.
-        // The array contains multiple arrays that are a series of points to draw fixes between.
-        // In this case, ENI->PYE, MXW->PYE, PYE->STINS->HADLY->OSI are the lines drawn.
-        // Additionally, you'll notice two asterisks ('*'). This is an optional flag that, if invoked for "FIXXX"
-        // will tell canvas.js to write "FIXXX.PYE1" next to FIXXX on the scope. If no such flags are present,
-        // then the ICAO identifier for the SID will be drawn at the first point of the "draw" array. For european-
-        // style STARs, where they always end at the fix for which the STAR is named, don't use the flags.
-    }
-  },
-  "departures": {
-    "airlines": [
-      ["three-letter ICAO airline code/fleet", 0], // see "Aircraft/Airline selectors" below
-      ...
-    ],
-    "destinations": [
-      "LISST", "OF", "SIDS", "ACRFT", "WILLL", "FLYYY", "TO"  // these must each be a defined SID above
+    "radio": {
+        "twr": "Las Vegas Tower",
+        "app": "Las Vegas Approach",
+        "dep": "Las Vegas Departure"
+    },
+    "icao": "KLAS",                             // Uppercase ICAO airport code.
```

Comment:
I know you're still working on this, but just a reminder, all these inline comments should moved out of the example to their own section below.  This example block should be data only, just like you would find if you opened up `klas.json` (though, obviously, abbreviated here)